qute.ls/com.redhat.qute.ls/src/test/java/com/redhat/qute/services/diagnostics/QuteDiagnosticsInExpressionWithNamespaceTest.java

```diff
@@ -58,8 +58,8 @@ public void validInjectMethodNamespace() {
 
 		template = "{config:propertyXXXX('qute')}";
 		testDiagnosticsFor(template,
-				d(0, 8, 0, 20, QuteErrorCode.UnkwownNamespaceResolverMethod,
-						"`propertyXXXX` cannot be resolved or is not a method of `config` namespace resolver.",
+				d(0, 8, 0, 29, QuteErrorCode.UnknownNamespaceResolverMethod,
+						"`getConfigPropertyXXXX` cannot be resolved or is not a method of `config` namespace resolver.",

```

Comment:
replace with (like master)

`"`propertyXXXX` cannot be resolved or is not a method of `config` namespace resolver.",`